src/app/pages/server-details/server-details.component.ts

```diff
@@ -1166,7 +1166,7 @@ export class ServerDetailsComponent implements OnInit {
     if(!isMulti) {
       return this.serverDetails.benchmark_scores?.find((b) => b.benchmark_id === 'stress_ng:cpu_all' && (b.config as any)?.cores === 1)?.score?.toFixed(0) || '-';
     } else {
-      return this.serverDetails.benchmark_scores?.find((b) => b.benchmark_id === 'stress_ng:cpu_all' && (b.config as any)?.cores !== 1)?.score?.toFixed(0) || '-';
+      return this.serverDetails.benchmark_scores?.find((b) => b.benchmark_id === 'stress_ng:cpu_all' && (b.config as any)?.cores === this.serverDetails.vcpus)?.score?.toFixed(0) || '-';
```

Comment:
I think those two are doing the same: when `isMulti` is true, then the config's `cores` is looked up against the server's `vcpus`.

Reply:
I think that's done .. in this PR, the updated line in the `server-compare.component.ts` is:

```py
return server.benchmark_scores?.find((b) => b.benchmark_id === 'stress_ng:cpu_all' && (b.config as any)?.cores === server.vcpus)?.score?.toFixed(0) || '-';
```

So that is also comparing with the number of VCPUs.